src/test/java/core/basesyntax/FruitTest.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Fruit;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitTest {
+    @Test
+    public void equals_fruitIsNotEqual_NotOk() {
+        Fruit fruit0 = new Fruit("apple");
+        Fruit fruit1 = new Fruit("banana");
```

Comment:
```suggestion
        Fruit fruitApple = new Fruit("apple");
        Fruit fruitBanana = new Fruit("banana");
```
Try not to use numbers in variable names